.github/ISSUE_TEMPLATE/02_feature_request.yml

```
@@ -0,0 +1,43 @@
+name: Feature Request
+description: Enchancement to gridify functionality
+title: "Feature Request:  <Insert Issue Title Here>"
+labels: ["enhancement", "programming"]
+assignees:
+  - polkas
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this feature request!  We love keeping gridify fresh!
+  - type: textarea
+    id: feature
+    attributes:
+      label: Feature Idea
+      description: Tell us your idea in as few words as possible
+      placeholder: "`derive_var_dtm` should do x, y and z"

```

We should probably change the placeholder to something that makes more sense
```suggestion
      placeholder: "`show_spec` should do x, y and z"
```